### ISSUE TITLE
Glover - HUD graphics issue at non-native resolution

### ISSUE THREAD

**Issue by Zinfidel** [gonetz/GLideN64#2622]
There are some 2D HUD issues at 4K:
![image](https://user-images.githubusercontent.com/1929934/146430713-c10f0fb5-272a-4e5b-a94c-fb9b5315099a.png)

I messed with the various texrec settings but none of them seemed to make a difference for this. The problem isn't present in Public Release 3.0. I've tested the most recent commit at the time of writing (7b507e74d4beb2e3565e749224c2b272c73d3e5e), GlideN64rev.abf81a52 (from the GoldenEye issue), and Public Release 4.0. All have the weird graphics.

Environment: Win 10, latest Mupen64-rr-lua, built-in RSP plug-in.

**Comment by Zinfidel** [comment]
Got some more info for this issue. On Public Release 3.0, with the option checked for native 2D graphics, the HUD problems go away, but the z-ordering of 2d elements gets messed up:
![GLideN64_Glover_001](https://user-images.githubusercontent.com/1929934/146612386-157e916f-1726-432a-91f9-6a4537446440.png)
If the native 2D graphics option is **unchecked**, the HUD elements again look like they do in the most recent version of the plugin, but the Z-ordering problem goes away.

And a bonus image I got while messing with video settings while the game was running on Public Release 4.0:
![GLideN64_Glover_000](https://user-images.githubusercontent.com/1929934/146612429-5b63ed59-14eb-44e8-be49-daef89933073.png)
I don't know that this image is valuable in any way but I found it interesting so who knows, might as well share it.

Again this all on the latest version of mupen64-rr-lua, default RSP plugin, running at 2880x2160.
